lesson16/part1/employee.py

```diff
@@ -0,0 +1,10 @@
+
+
+class Employee:
+
+    name:str
```

Comment:
It is better to create class fields private or at least protected if we do not want to provide an access from outside.